.github/workflows/prettier.yml

```diff
@@ -0,0 +1,39 @@
+name: Format workspace
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '**/*.md'
+      - '**/*.js'
+      - '**/*.ts'
+      - '**/*.tsx'
+      - '**/*.mts'
+      - '**/*.mjs'
+      - '**/*.graphql'
+
+permissions:
+  contents: read
+  pull-requests: read
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  prettier:
+    runs-on: buildjet-8vcpu-ubuntu-2204
+    steps:
+      - name: Get sources
+        uses: actions/checkout@v3
+
+      - name: Run prettier on cli/
+        shell: bash
+        run: |
+          cd cli
+          npx prettier --check .
```

Comment:
We should move towards formatting the entire repo, but there are configuration differences between cli/ and packages/, it will take some work to fix.